docs/_source/_common/tabs/argilla_install_python.md

```diff
@@ -0,0 +1,25 @@
+First things first! You can use a basic `pip install` to install for `pypi` but we also provide some custom extras and more developer-related installment options.
+
+::::{tab-set}
+
+:::{tab-item} Without extras
+```bash
+pip install argilla
+```
+:::
+
+:::{tab-item} With extras
+```
+pip install argilla[listeners] # use argilla.listeners background processes
+pip install argilla[server] # running FastAPI locally
+pip install argilla[postgresql] # use PostgreSQL instead of SQLite as database
```

Comment:
For safety I'd add double-quotes surrounding `argilla[...]` as e.g. `pip install "argilla[postgresql]"`